shell/lint.sh

```diff
@@ -13,6 +13,7 @@ if [ $# -ne 0  ]
     files=$@
 fi
 
+#verify isort
```

Comment:
nit: I think we can get rid of these comments as the code is pretty self-descriptive

Reply:
Okay. I will remove these comments.